src/Pim/Bundle/EnrichBundle/Connector/Processor/QuickExport/ProductProcessor.php

```diff
@@ -97,7 +98,12 @@ public function process($product)
         $productStandard = $this->normalizer->normalize($product, 'standard', $normalizerContext);
 
         if ($this->areAttributesToFilter($parameters)) {
-            $productStandard = $this->filterProperties($productStandard, $parameters->get('selected_properties'));
+            $selectedProperties = $parameters->get('selected_properties');
+            if (in_array('identifier', $selectedProperties)) {
+                $identifer = $this->attributeRepository->findOneBy(['type' => AttributeTypes::IDENTIFIER]);
+                $selectedProperties = array_merge([$identifer->getCode()], $selectedProperties);
```

Comment:
You could have used http://php.net/manual/en/function.array-unshift.php if you wanted also.